ingest/rules/nextclade.smk

```diff
@@ -23,24 +23,24 @@ DATASET_NAMES = config["nextclade"]["dataset_name"]
 wildcard_constraints:
     DATASET_NAME = "|".join(DATASET_NAMES)
 
-# rule get_nextclade_dataset:
-#     """Download Nextclade dataset"""
-#     output:
-#         dataset=f"data/nextclade_data/{DATASET_NAME}.zip",
-#     params:
-#         dataset_name=DATASET_NAME
-#     shell:
-#         r"""
-#         nextclade3 dataset get \
-#             --name={params.dataset_name:q} \
-#             --output-zip={output.dataset} \
-#             --verbose
-#         """
-
+rule get_nextclade_dataset:
+    """Download Nextclade dataset"""
+    output:
+        dataset="data/nextclade_data/{DATASET_NAME}.zip",
+    params:
+        # Specifically pull datasets from nextstrain/mumps
+        dataset_name=lambda wildcards: f"nextstrain/mumps/{wildcards.DATASET_NAME}"
```

Comment:
_nit_

I'd document that we've hardcoded `nextstrain/mumps/` here in the [config](https://github.com/nextstrain/mumps/blob/af0091a79f5320a13bd1c9ae1cd13ae7250e5542/ingest/defaults/config.yaml#L127-L129)

E.g.
```yaml
nextclade:
  # The name of the Nextclade dataset to use for running nextclade.
  # Run `nextclade dataset list --search nextstrain/mumps` to get a list of available mumps datasets
  # The dataset names listed should match `nextstrain/mumps/<dataset_name>`
  dataset_name: ["sh","genome"]
```